app/uk/gov/hmrc/helptosave/services/EligibilityStatsService.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.services
+
+import cats.instances.int._
+import cats.instances.option._
+import cats.instances.string._
+import cats.syntax.eq._
+import com.google.inject.{ImplementedBy, Inject, Singleton}
+import uk.gov.hmrc.helptosave.metrics.Metrics
+import uk.gov.hmrc.helptosave.repo.EligibilityStatsStore
+import uk.gov.hmrc.helptosave.util.{LogMessageTransformer, Logging}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@ImplementedBy(classOf[EligibilityStatsServiceImpl])
+trait EligibilityStatsService {
+
+  def getEligibilityStats()(implicit ec: ExecutionContext): Future[Either[String, String]]
+
+}
+
+@Singleton
+class EligibilityStatsServiceImpl @Inject() (eligibilityReportStore: EligibilityStatsStore,
+                                             metrics:                Metrics)(implicit ninoLogMessageTransformer: LogMessageTransformer)
+  extends EligibilityStatsService with Logging {
+
+  override def getEligibilityStats()(implicit ec: ExecutionContext): Future[Either[String, String]] = {
```

Comment:
In order to define the separation of concerns better, it makes more sense for this service to return [Future[EIther[String, List[EligibilityStats]]]` and for the `EligibilityStatsHandler` to look like:
```scala
trait EligibilityStatsHandler {
  def handleStats(result: List[EligibilityStats]): Unit
}
```
In this way, the service is in charge of getting the stats - or you may not even need the service if the actor can get an instance of the repo directly. And then the actor is in charge getting the stats and handling any errors and the handler is in charge of handling the stats (by logging them).